rust/core-lib/src/file.rs

```diff
@@ -136,12 +178,23 @@ impl FileManager {
     fn save_new(&mut self, path: &Path, text: &Rope, id: BufferId) -> Result<(), FileError> {
         try_save(path, text, CharacterEncoding::Utf8)
             .map_err(|e| FileError::Io(e, path.to_owned()))?;
+
+        #[cfg(feature = "notify")]
+        let new_tail_details = TailDetails {
+            current_position_in_tail: 0,
```

Comment:
I don't think we should need to track this explicitly? Unless I'm misunderstanding, this should always be the same as the current length of the text we have loaded?

Reply:
Correct. It should be length of text.